### PR TITLE
Convert help to tooltip (AdditionalInfo)

### PR DESCRIPTION
**Pull request by cehsu**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5452#event-1328241754
screen shot 2017-11-07 at 2 51 36 pm
screen shot 2017-11-07 at 2 51 52 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@cehsu I think Peggy meant the pattern in the `AdditionalInfo` component, not the collapsible panel.

**Comment by cehsu** [comment]
@jbalboni Thanks for pointing me toward the appropriate component. I will use that instead.

**Comment by stacyhlinn** [comment]
@cehsu I also want to make sure that the "learn more" additional info is going to appear above the radio buttons. Thanks!

**Comment by cehsu** [comment]
Thanks @jbalboni @peggygannon @stacyhlinn for the comments, kindly refer to the updated screenshots above at your convenience.

**Comment by cehsu** [comment]
Thanks @jbalboni @peggygannon @stacyhlinn for the feedback. Kindly refer to the updated screenshot at your convenience. @jbalboni I have used `ui:description` in a hacky way, to achieve the desired placement. Should we consider looking into changing the default `ui:help` behavior in the future?

**Comment by jbalboni** [comment]
@cehsu I think that's a good idea, to use `ui:help`. It shouldn't be too hard to set up, you would add it to `FieldTemplate`.

**Comment by jbalboni** [comment]
@cehsu I think the way you have it for `ui:description` works fine. That's what that field is meant for.

File: src/js/hca/config/form.js [hunk 1]
```diff
@@ -729,12 +730,16 @@ const formConfig = {
             isMedicaidEligible: {
               'ui:title': 'Are you eligible for Medicaid?',
               'ui:widget': 'yesNo',
-              'ui:help': 'Medicaid is a United States health program for eligible individuals and families with low income and few resources.'
+              'ui:description': <AdditionalInfo triggerText="Learn more about Medicaid.">
```
Comment (jbalboni) [inline src/js/hca/config/form.js]: I would be interested to see what this looks like when there's an error with the field.

File: src/js/hca/config/form.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
Comment (jbalboni) [inline src/js/hca/config/form.js]: Should change this file to jsx or move the description element into helpers.

**Review by jbalboni — APPROVED** [review]
LGTM, had a couple questions.

File: src/js/hca/config/form.js (hunk 2 above)
Comment (cehsu) [reply src/js/hca/config/form.js]: @jbalboni great point, change made.

File: src/js/hca/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/js/hca/config/form.js]: Great point @jbalboni, screenshot provided.

**Review by peggygannon — COMMENTED** [review]
@ceshu. I'm not sure we need the "Please provide response" line that is shown in the first screenshot. We already have (*Required) next to the question. @stacyhlinn What do you think? 
Otherwise it looks good!

**Comment by stacyhlinn** [comment]
@peggygannon I think what we're seeing here is when a user tries to hit continue without answering this required question, and the page is showing validation error. 
This looks good to me!

**Review by stacyhlinn — APPROVED** [review]

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
@stacyhlinn OK, thanks! looks good.

**Pull request merged**